apps/server/test/modules/management/database.e2e-spec.ts

```diff
@@ -2,15 +2,16 @@ import { Test, TestingModule } from '@nestjs/testing';
 import { INestApplication } from '@nestjs/common';
 import request from 'supertest';
 import { MikroORM } from '@mikro-orm/core';
-import { ManagementModule } from '../../../src/modules/management/management.module';
+import { ManagementTestModule } from '../../../src/modules/management/management.module';
```

Comment:
Please use named paths. This should be `@src/modules/management/management.module`.